tests/controllers/UsersControllerTest.php

```diff
@@ -226,4 +226,192 @@ public function testThatUserDataIsNotDoubleEscapedOnUserEdit()
 
         $controller->updateUser($request, $db);
     }
+
+    /**
+     * Ensures that if the setTrusted method is called and no user_id is set,
+     * an exception is thrown
+     *
+     * @return void
+     *
+     * @test
```

Comment:
Personally I normally use the prefix over the annotation, but most of the already existing tests there is both, so I've gone with both for consistency